webapp/src/app.css

```diff
@@ -78,6 +80,50 @@ pre {
     margin: 0 auto;
 }
 
+.logo-small {
+    border-radius: 2rem;
+    max-width: 12rem;
+}
+
+.logo-medium {
+    border-radius: 2rem;
+    max-width: 15rem;
+}
+.submit-button {
+    background-color: #070707;
+    color: white;
+    border-radius: 4px;
+    font-size: 1rem;
+    min-width: 6rem;
+    min-height: 2rem;
+    margin: 0.5rem 0 0.5rem 0;
+    cursor: pointer;
+    transition: background-color 0.3s;
+}
+
+.auth-form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-items: center;
+    width: auto;
```

Comment:
Default values for height and width are auto so these aren't needed to be explicitly written.